mkdocs.yml

```diff
@@ -57,6 +58,9 @@ extra_css:
 
 plugins:
   - search
+  - mkdocs-video
+  - redirects
+  - exclude
```

Comment:
Are those actually used/planned to be used or is it copied from docs.dasharo without any consideration?

Reply:
Copied from docs.dasharo to synchronize both repos. @macpijan that's what I was supposed to do, right?

Reply:
I removed other plugins and also additional ones that from what I can see are not used anywhere.